azure-pipelines/official.yml

```diff
@@ -4,6 +4,18 @@ trigger:
   paths:
     exclude: ["*.md"]
 
+schedules:
+  - cron: "0 8 23-29 * 0"
+    displayName: "Monthly build check"
+    branches:
+      include: 
+        - main
+        - rel/*
```

Comment:
`rel/*` is a bad idea as it exists today.  We didn't use rel branches regularly, and those that do exist are super outdated in terms of build infrastructure so the builds will almost certainly fail.

I would stick with just main until another branching strategy becomes common.